lib/eblox/data/post.ex

```diff
@@ -17,6 +17,29 @@ defmodule Eblox.Data.Post do
 
   use Finitomata, @fsm
 
+  defmodule Properties do
+    @moduledoc """
+    Post properties extracted from provider or content.
+    """
+
+    alias Eblox.Data.Post.Properties
+
+    @type t :: %{
+            __struct__: Properties,
+            links: list(term()),
+            tags: list(binary())
+          }
+    defstruct links: [], tags: []
```

Comment:
```suggestion
    defstruct links: MapSet.new(), tags: MapSet.new()
```
This would be easier to merge _and_ faster to retrieve.